packages/flutter_tools/pubspec.yaml

```diff
@@ -47,7 +47,6 @@ dependencies:
   mime: 0.9.7
   logging: 0.11.4
   http_multi_server: 2.2.0
-  devtools_server: 0.9.5
```

Comment:
There is a whole song-and-dance to remove a dependency from this list, it is probably easier to leave it here and we can clean it up later.

Reply:
reverted